src/sentry/tasks/post_process.py

```diff
@@ -699,7 +699,14 @@ def run_post_process_job(job: PostProcessJob):
 
     for pipeline_step in pipeline:
         try:
-            with sentry_sdk.start_span(op=f"tasks.post_process_group.{pipeline_step.__name__}"):
+            with metrics.timer(
+                "sentry.tasks.post_process.run_post_process_job.pipeline.duration",
```

Comment:
QQ: isn't `sentry.` prefix automatically prepended?

Reply:
oh, I didn't realize that. There are few places in this file still using "sentry.tasks.post_process" for `metrics.incr`. I can suppress prefix for sure